manifests/0000_90_openshift-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -65,7 +65,7 @@ spec:
       sourceLabels:
       - __name__
     - action: drop
-      regex: apiserver_request_duration_seconds_bucket;(0.15|0.25|0.3|0.35|0.4|0.45|0.6|0.7|0.8|0.9|1.25|1.5|1.75|2.5|3|3.5|4.5|6|7|8|9|15|25|30|50)
+      regex: apiserver_request_duration_seconds_bucket;(0.15|0.25|0.3|0.35|0.4|0.45|0.6|0.7|0.8|0.9|1.25|1.5|1.75|2.5|3|3.5|4.5|6|7|8|9|15|25|30|50)(\.0)?
```

Comment:
I think that this reg pattern allows an optional .0 at the end. Am I right ?

Reply:
For example `apiserver_request_duration_seconds_bucket;30.0`

Reply:
You're right.
In Prometheus v3, the values are floats and we should account for that, more details in https://issues.redhat.com//browse/MON-4129